project/plugins/google.py

```diff
@@ -66,9 +76,20 @@ def wait_for_operation(compute, project, region, operation):
 
         time.sleep(5)
 
+
 def rotate_instance_groups(configMap, username,  **key_args):
     auth = configMap['Global']['google_credentials']['client_cred']
+    regions = key_args.get('regions')
+    rotate_gcp_instance_group(auth, regions, 2)
+
+
+def rotate_fg_instance_groups(configMap, username,  **key_args):
```

Comment:
Is this called from anywhere? Can't we just modify the rotate_instance_groups method to pass in the credential you are looking for (client_cred or fg_cred) instead of creating a new very specific method with fg in the name?

Reply:
Ah yes, I picked this way to not make any changes to the existing rotate_instance_groups() and its counterparts in the config file.  Since both functions require different credentials and variable max_unavailable. I wasn't sure which way was better and I want to minimize my change to the existing config.yaml file as possible so I picked this way.